addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -255,9 +255,12 @@ def _sync_microsoft2odoo(self, microsoft_events: MicrosoftEvent, default_reminde
             dict(self._microsoft_to_odoo_values(e, default_reminders, default_values), need_sync_m=False)
             for e in (new - new_recurrent)
         ]
-        new_odoo = self.create(odoo_values)
+        new_odoo = self.with_context(dont_notify=True).create(odoo_values)
 
-        synced_recurrent_records = self._sync_recurrence_microsoft2odoo(new_recurrent)
+        synced_recurrent_records = self.with_context(dont_notify=True)._sync_recurrence_microsoft2odoo(new_recurrent)
+        if not self._context.get("dont_notify"):
```

Comment:
you should avoid negative variable names
```suggestion
        if self._context.get("notify_attendees", True):
```

Reply:
The point is to use the context flag `dont_notify` from the original code.